### ISSUE TITLE
No more logs

### ISSUE THREAD

**Issue by JackFrost67** [RepRapLtd/Electric3DPrinter#2]
Hi,
I was interested in this new design for a new concept of 3D printing. I recently run into Berkeley design, in which I have lots of dubts (like the way the uncured resin is treated, because all the resin is hit by a small part of UV light).
I discovered your design, but the recent log is from 2019, but how is it going?

**Comment by RepRapLtd** [comment]
I'm afraid that we haven't had much time to work on this more lately, but we hope to return to it soon.  Of course, if anyone else wants to move it forward...

**Comment by JackFrost67** [comment]
yeah, I started breaking things down to better understand how the simulation works. Do you mind writing down what you were thinking to do next or bugs, etc...?
